catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -95,6 +96,7 @@ type Props = {|
   iiifImageLocationLicenseId: ?LicenseType,
   encoreLink: ?string,
   useBetaDownloadComponent: boolean,
+  showSingleImageWorkPreview: boolean,
```

Comment:
We can use the `TogglesContext` for this, but whatever is easiest.
I've just found it keeps the concerns closer to where we need them, and stops passing props about.

Reply:
I've got some other branches, branched off of this - will deal with this in those to limit conflicts